common/buildings/00_duchy_capital_buildings.txt

```diff
@@ -341,6 +341,9 @@ siege_works_02 = {
 	character_modifier = {
```

Comment:
Ok, so one dev mentioned it in the dev channel, so I'll bring it here as a review.
There's the "taur" problem. Some races have 4 legs. Obviously, they can't ride anything. They're basically cavalry themselves. I see only one solution:
- Block innovations introducing new cavalry units like camels and elekks.
- Relocalize their buildings if it's needed (if it mentions riding on them).

I've added `taur_culture_trigger` to represent such cultures, you may use it for these restrictions.

Reply:
Sent an image in dev, tol'vir isn't working for me.  The trigger is working in general though, I tested with Dragons and Dryads.